### ISSUE TITLE
Driver [ovh] is not supported.

### ISSUE THREAD

**Issue by lastonoga** [sausin/laravel-ovh#1]
Providers in app config

```
'providers' => [

        /*
         * Laravel Framework Service Providers...
         */
        Illuminate\Auth\AuthServiceProvider::class,
        Illuminate\Broadcasting\BroadcastServiceProvider::class,
        Illuminate\Bus\BusServiceProvider::class,
        Illuminate\Cache\CacheServiceProvider::class,
        Illuminate\Foundation\Providers\ConsoleSupportServiceProvider::class,
        Illuminate\Cookie\CookieServiceProvider::class,
        Illuminate\Database\DatabaseServiceProvider::class,
        Illuminate\Encryption\EncryptionServiceProvider::class,
        Illuminate\Filesystem\FilesystemServiceProvider::class,
        Illuminate\Foundation\Providers\FoundationServiceProvider::class,
        Illuminate\Hashing\HashServiceProvider::class,
        Illuminate\Mail\MailServiceProvider::class,
        Illuminate\Pagination\PaginationServiceProvider::class,
        Illuminate\Pipeline\PipelineServiceProvider::class,
        Illuminate\Queue\QueueServiceProvider::class,
        Illuminate\Redis\RedisServiceProvider::class,
        Illuminate\Auth\Passwords\PasswordResetServiceProvider::class,
        Illuminate\Session\SessionServiceProvider::class,
        Illuminate\Translation\TranslationServiceProvider::class,
        Illuminate\Validation\ValidationServiceProvider::class,
        Illuminate\View\ViewServiceProvider::class,
        // Drauta\OvhObjectStorage\OvhServiceProvider::class,
        
        Krlove\EloquentModelGenerator\Provider\GeneratorServiceProvider::class,
        NotificationChannels\Telegram\TelegramServiceProvider::class,
        Approached\LaravelImageOptimizer\ServiceProvider::class,
        Sausin\LaravelOvh\OVHServiceProvider::class,
        /*
         * Application Service Providers...
         */
        App\Providers\AppServiceProvider::class,
        App\Providers\AuthServiceProvider::class,
        App\Providers\EventServiceProvider::class,
        App\Providers\RouteServiceProvider::class,

    ],
```

Test code in controller

`\Storage::disk('ovh')->put('test-ovh', 'test');`

filesystem.php config

```
'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'visibility' => 'public',
        ],

        's3' => [
            'driver' => 's3',
            'key' => 'your-key',
            'secret' => 'your-secret',
            'region' => 'your-region',
            'bucket' => 'your-bucket',
        ],

        'ovh' => [
            'server' => 'https://auth.cloud.ovh.net/v2.0/',
            'driver' => 'ovh',
            'user' => 'aDfgA3heJpM',
            'pass' => '1D78asdDa9860Fa6D67daAsd56asD7A85d',  
            'region' =>'SBG1',
            'tenantName' => '1916123345237',
            'container' => 'public',
            'projectId' => '23asd54asd3a45sd3a5sd',
        ],

    ],
```

And still there is an error 

Driver [ovh] is not supported.

**Comment by lastonoga** [comment]
In different library connection and providers work

**Comment by sausin** [comment]
Run this command in your project root:

`composer dump-autoload`

and try again. 

Also, if you can let me know the version of laravel and this library you're using in case it's still not working

Regards,
Saurabh

**Comment by lastonoga** [comment]
Thanks, but it still doesn't work.

Laravel - 5.4
Library - 0.5.1

Its magic.. I tried to see if Storage::extend starts and its not. Anyway i put class in app config

**Comment by sausin** [comment]
Not sure what is happening in your case. I just created a new laravel 5.4 project and it works fine (though I used the latest 0.6.0 version, but for this test it should be same). 

Copying your driver config as-is in the filesystem results in a 401 (obviously, as the credentials are incorrect) but not that the driver is not supported.

I am not sure how to help if I can't reproduce your issue :confused:
